mlflow/gateway/__init__.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+import pathlib
+import signal
+import sys
+import time
+from typing import Optional
+
+from mlflow.exceptions import MlflowException
+from mlflow.gateway.constants import CONF_PATH_ENV_VAR, SERVER_HOUSEKEEPING_PAUSE
+from mlflow.protos.databricks_pb2 import BAD_REQUEST
+
+import subprocess
+
+server_process: Optional[subprocess.Popen] = None
+gateway_host: Optional[str] = None
+gateway_port: Optional[int] = None
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.DEBUG)
+_stream_handler = logging.StreamHandler()
+_stream_handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+_stream_handler.setFormatter(formatter)
+_logger.addHandler(_stream_handler)
+_logger.propagate = False
```

Comment:
Can we reuse the MLflow logging format? Do we need to set these confs for uvicorn compatibility or something similar? If not, would be great to reuse existing configs / logging infra

Reply:
definitely. The original intention was to add a little distinctness and additional precision in logged timestamps of events, but now that I'm looking at it again it just doesn't really seem worth it. Sticking with the standard logger in DEBUG mode for server init events.